abap-code-review/src/introduction.adoc

```diff
@@ -0,0 +1,40 @@
+[cols="1,3",frame=none,grid=none]
+|===
+|Home
+|link:https://github.com/SAP/styleguides[https://github.com/SAP/styleguides]
+
+|License
+|link:https://github.com/SAP/styleguides/blob/main/LICENSE[Creative Commons]
+
+|Build
+|{docdatetime}
+|===
+
+== Introduction
+
+Code review is a measure for assuring software quality by letting developers read the code of others. While the primary goal is to increase quality by finding defects, it also fosters learning and shared code ownership.
+
+link:https://help.sap.com/viewer/Continuous-Integration-and-Delivery-Introduction-Guide/5ba483a2c97b4ad5ab0148f4a6c5a9ee.html[Continuous integration / continuous delivery] (CI/CD) is commonly seen as a prerequisite to faster delivery of cloud-based products. This comprises the ability to review, test, and, if required, veto changes before they affect every other developer working on the same repository.  This capability to validate a change in _isolation_ before integrating it is also known as link:https://trunkbaseddevelopment.com/game-changers/#google-s-internal-devops-2006-onwards[proposed commits] or link:https://martinfowler.com/bliki/PendingHead.html[pending head]. For instance, most git platforms have implemented this concept as depicted below.
+
+[plantuml,activity,svg,align="center"]
+....
+@startuml
+skinparam monochrome true
+
+start
+:Request;
+repeat
+  :Automated Review;
```

Comment:
Are you sure you want to call these code analysis steps "Automated Review"? The word Review somehow sounds inappropriate here.

Reply:
Done.  Changed to "Automated Checks"